assets/src/dashboard/utils/index.js

```diff
@@ -24,3 +24,4 @@ export { default as useDashboardResultsLabel } from './useDashboardResultsLabel'
 export { default as usePagePreviewSize } from './usePagePreviewSize';
 export { default as useStoryView } from './useStoryView';
 export { default as throttleToAnimationFrame } from './throttleToAnimationFrame';
+export { default as useResizeEffect } from '../../edit-story/utils/useResizeEffect';
```

Comment:
Thoughts about this structure for when we reference edit-story utils instead of copy and paste? I think this makes it clear where the codeshare is.